assets/xml/objects/object_eg.xml

```diff
@@ -1,106 +1,140 @@
 ﻿<Root>
     <File Name="object_eg" Segment="6">
```

Comment:
Since this is fully documented, can you put a description of its comments above the \<File\> node? You can check out a bunch of objects to see an example of this; I'll just toss out object_boss02.xml as an example.

Reply:
Added the descriptions. What do you think?

Reply:
Looks good outside of a typo: `Assets` instead of `Assests` (there's an extra "s" in the latter one).